src/main/java/tictactoe/Grid.java

```diff
@@ -40,4 +40,15 @@ public String setSquare(int index, String data) {
     public int getSize() {
         return squares.size();
     }
+
+    public Grid copyGrid() {
+        var squares = copySquares();
```

Comment:
I think it's a little confusing that this `squares` variable shadows the field named `squares`. It's legitimate code and it works (I'd access the field with `this.squares`). Do you think it would be clearer if the variable was named something else?

Reply:
Easiest way around this is: alt + command + N to inline the reference :)

Reply:
Thanks for spotting this, Ashley, and thanks for the tip, Matt. I'll change this :)